go.sum

```diff
@@ -1,15 +1,60 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
I really really really really really don't like this. This is 3xing the number of dependencies. Is there another option?